Dockerfile-cli.template

```diff
@@ -32,16 +31,6 @@ RUN set -ex; \
 	apk add --virtual .wordpress-phpexts-rundeps $runDeps; \
 	apk del .build-deps
 
-# set recommended PHP.ini settings
```

Comment:
This comment line still applies (and I wonder if we should maybe leave in a comment about opcache being intentionally left out here so we don't accidentally add it back again next time we do a `vimdiff` to the `cli` template 😅😇).